common/primitives/src/messages.rs

```diff
@@ -257,9 +257,9 @@ mod tests {
 			&request,
 		);
 		// NOT FULL
-		assert_eq!(false, is_full);
+		assert!(!is_full);
```

Comment:
Ok. I didn't like this until I learned about how rust is improving the `assert!` macro. See https://github.com/rust-lang/rust/issues/44838 for more info. Although not all of it works now, it is the future.